### PR TITLE
Adding Get3dsAvailability into BinLookup directory

### PR DESCRIPTION
**Pull request by fabricioblz**
**Description**
In order to make it possible to call `/get3dsAvailability`.
This PR make this possible via SDK

**Usage**

```php
        $json = '{
          "cardNumber": "4212345678901245",
          "merchantAccount": "TestMerchant"
        }';

        $params = json_decode($json, true);
        $service = new \Adyen\Service\BinLookup($this->apiClient)

        return json_encode($service->get3dsAvailability($params));

```

### CODE REVIEW

**Comment by Aleffio** [comment]
Hi @fabricioblz 

Thank you for the PR!
We are going to review it and merge it.

Kind Regards,
Alessio,
Adyen

**Comment by fabricioblz** [comment]
Thank you for the review.

I've made the modifications. Could you check please?

File: src/Adyen/Service/ResourceModel/BinLookup/Get3dsAvailability.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    protected $allowApplicationInfo = true;
+
+    /**
+     * Authorise constructor.
+     *
+     * @param \Adyen\Service $service
+     */
+    public function __construct($service)
+    {
+        $this->endpoint = $service->getClient()->getConfig()->get('endpoint') . '/pal/servlet/BinLookup/' . $service->getClient()->getApiBinLookupVersion() . 'get3dsAvailability';
```
Comment (cyattilakiss) [inline src/Adyen/Service/ResourceModel/BinLookup/Get3dsAvailability.php]:
```suggestion
        $this->endpoint = $service->getClient()->getConfig()->get('endpoint') . '/pal/servlet/BinLookup/' . $service->getClient()->getApiBinLookupVersion() . '/get3dsAvailability';
```

Sorry I missed a slash sign.
Everything else looks good 👍

**Comment by fabricioblz** [comment]
No worries, I commited your change = )

File: src/Adyen/Service/ResourceModel/BinLookup/Get3dsAvailability.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+     * @var string
+     */
+    protected $endpoint;
+
+    /**
+     * Include applicationInfo key in the request parameters
+     *
+     * @var bool
+     */
+    protected $allowApplicationInfo = true;
```
Comment (rikterbeek) [inline src/Adyen/Service/ResourceModel/BinLookup/Get3dsAvailability.php]: could you put this to false as we do not support it for this endpoint.

Comment (fabricioblz) [reply src/Adyen/Service/ResourceModel/BinLookup/Get3dsAvailability.php]: Sure!

**Comment by fabricioblz** [comment]
Now is every thing all right? 

Let me know in case of any additional change.

**Comment by rikterbeek** [comment]
Hi @fabricioblz,

Thanks for the PR looks great. Are you able to add a test case for this as well ?

Regards,
Rik
Adyen

**Comment by fabricioblz** [comment]
Sure @rikterbeek,

I've made two basic tests:

**Comment by cyattilakiss** [comment]
Hi @fabricioblz ,
We are going to release your changes in the next library release. Thank you for your contribution.
Best,
Attila
Adyen

**Comment by fabricioblz** [comment]
Great! Thank you!

**Pull request merged**